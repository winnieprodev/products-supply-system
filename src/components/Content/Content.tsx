import React from "react";
import { Empty } from "utils/types";

import * as Styled from "./content.styled";

const Content = ({ children }: React.PropsWithChildren<Empty>) => (
  <Styled.Content>{children}</Styled.Content>
);

export default React.memo(Content);
