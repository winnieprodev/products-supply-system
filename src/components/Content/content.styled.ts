import styled from "styled-components";

export const Content = styled.div`
  width: 80%;
  min-width: 1024px;
  max-width: 1600px;
  margin: 36px auto;
  padding: 16px;
`;
