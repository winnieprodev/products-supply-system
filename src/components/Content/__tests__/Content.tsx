import { render } from "@testing-library/react";

import Content from "../Content";

describe("Content", () => {
  describe("Snapshot tests", () => {
    test("it matches the snapshot", () => {
      const { container } = render(<Content>test</Content>);
      expect(container.firstChild).toMatchSnapshot();
    });
  });
});
