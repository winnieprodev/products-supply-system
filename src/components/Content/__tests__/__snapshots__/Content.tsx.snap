// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Content Snapshot tests it matches the snapshot 1`] = `
.c0 {
  width: 80%;
  min-width: 1024px;
  max-width: 1600px;
  margin: 36px auto;
  padding: 16px;
}

<div
  class="c0"
>
  test
</div>
`;
